src/sparseml/onnx/utils/model.py

```diff
@@ -40,7 +40,7 @@
     get_prunable_node_from_foldable,
     is_foldable_node,
 )
-from sparsezoo.objects import File, Model
+from sparsezoo import File, Model
```

Comment:
these classes are not 1:1 interchangeable - do we account for this in this file?

Reply:
Definitely, something that I cannot say was 100% tested.